modules/components/src/NestedTreeView/NestedTreeView.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ReactTreeView from './ReactTreeView';
+import './style.css';
+
+const NestedTreeView = ({ dataSource }) =>
+  dataSource.map(
+    (node, i) =>
+      node.children ? (
+        <ReactTreeView
+          key={i}
```

Comment:
Can you make the key something like `node.key` or `node.key || node.title` if we don't want to force a key field.